File: POOBanca/PHP/Livro.php
<?php
require_once "Produto.php";

class Livro extends Produto {
    public function __construct($nome, $editora, $totPaginas) {
        parent::__construct($nome, $editora, $totPaginas);
    }

    public function __destruct() {
        echo "<p>Objeto Livro Destruído com Sucesso!</p>";
    }
}

File: POOBanca/C++/Livro.cpp
#include "Livro.h"

Livro::Livro(string nome, string editora, int totPaginas) : Produto(nome, editora, totPaginas) {

}

Livro::~Livro() {
    cout << "Objeto Livro Destruído com Sucesso!" << endl;
}
